pkg/model/manifest.go

```diff
@@ -85,6 +85,11 @@ func (m Manifest) IsLocal() bool {
 	return ok
 }
 
+func (m Manifest) WithLocalTarget(lTarget LocalTarget) Manifest {
```

Comment:
can't you use `WithDeployTarget` instead? (`LocalTarget` implements `TargetSpec`)

Reply:
Yup, good call!